core/libp2p/multi/multibase_codec/codecs/base16.cpp

```diff
@@ -18,19 +19,17 @@ namespace {
    */
   bool encodingCaseIsUpper(std::string_view string) {
     return std::all_of(string.begin(), string.end(), [](const char &c) {
-      return !std::isalpha(c) || static_cast<bool>(std::isupper(c)); // NOLINT
+      return !std::isalpha(c) || static_cast<bool>(std::isupper(c));  // NOLINT
```

Comment:
By the way, which error appears, if you remove NOLINt? Can't it be fixed? Less NOLINTs, the better

Reply:
Er, it's not my code so I'm not sure. But I can try.